components/contactsList/index.jsx

```diff
@@ -134,24 +137,44 @@ function ContactsList() {
         ]}
       />
       <div className={containerClass}>
-        <table className={bem("table")}>
-          <tbody className={bem("table__body")}>
-            {contacts.map(({ name, avatar }) => (
-              <tr className={bem("table__body-row")}>
-                <td
-                  className={clsx(
-                    bem("table__body-cell"),
-                    bem("table__body-cell--align-end"),
-                    bem("table__body-cell--width-preview")
-                  )}
-                >
-                  <Avatar className={bem("avatar")} alt={name} src={avatar} />
-                </td>
-                <td>{name}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
+        <Table
+          things={contacts}
+          className={bem("table")}
+          // prettier-ignore
+          ascIndicator={(
+            <span role="img" aria-label="Sorted in ascending order">
```

Comment:
I think it'd be nice to pull the indicators into their own components, so we can re-use them as we build more tables.

Reply:
I second this - it should be a small component in Prism React components.